doc/development/deprecations.rst

```diff
@@ -105,6 +99,12 @@ for details on how to port your legacy code to the new system. The following fun
 Completed deprecation cycles
 ----------------------------
 
+* The `qml.gradients.hamiltonian_grad` function has been removed.
+  This gradient recipe is not required with the new operator arithmetic system.
```

Comment:
```suggestion
* The `qml.gradients.hamiltonian_grad` function has been removed because this gradient recipe is no
  longer required with the :doc:`new operator arithmetic system </news/new_opmath>`.
```